contributing.md

```diff
@@ -0,0 +1,141 @@
+# Contributing
+
+> You want to contribute to the website? Welcome!
+> You will find here resources and advices that could help you.
+
+If you have any questions, you can ask them in #website-dev channel in our Discord.
+
+ * [Preparing the development environment](#preparing-the-development-environment)
+ * [Guidelines](#guidelines-1)
+ * [Localization](#localization)
+
+<!-- /code_chunk_output -->
+
+
+## Preparing the development environment
+### Fork the project
+* Fork the repo.
+* **Always create a new branch before working on your fork.**
+* You can now improve the website.
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
+
+When your done submit a pull request.
+
+### Running the website
+> When you make modifications to the site it can be useful to render the code. You will thus see how the modifications look like.
+
+See [Building local enviroment](https://github.com/Paperback-iOS/website#building-local-enviroment)
+
+## Guidelines
+### Glossary
+
+ * App/application --> app
+ * Device/iPhone/iPad/phone --> iPhone or iPad
+ * iOS --> iOS or iPadOS
+
+> Be careful of capitalization:
+
+* Paperback
+* MangaDex
+
+* AltStore
+* AltServer
+
+* Safari
+* macOS
+
+### Syntax and layout
+> To create a numbered list use `1.` on all lines:
+```markdown
+1. Line 1
+1. Line 2
+```
+
+> Write `.IPA` in a code block.
+
+> Use `[Image]()` or `[GIF]()` to link to an explicative picture.
+
+### "Not up to date" warning
+> You've made a major modification to the English version of the site and you want to show that a translation is not up to date?
+> You can  mark a page as "not up to date" by adding the following code at the top of the concerned page.
+
+```html
+<notuptodate/>
+```
+It will create a translated warning saying:
+> Major changes have been made in the English version of this page. We suggest you to use this one.
```

Comment:
I understand that the warning isn't defined in this page, although the wording seems strange.

```suggestion
> Major changes have been made in the English version of this page. We suggest you use this revision instead.
```

Reply:
I used your more detailed suggestion in d708621:
```
Major changes have been made in the English version of this page. We suggest you use the English revision until this page is up to date.
```